src/main/java/Task.java

```diff
@@ -0,0 +1,9 @@
+public class Task {
+    public String taskName;
```

Comment:
Variables should not be public

Reply:
Yes, I will make those private and use getter and setter functions instead

Reply:
Yes, I agree. I also made this mistake in my coding.